internal/cri/config/config.go

```diff
@@ -574,11 +574,6 @@ func CheckLocalImagePullConfigs(ctx context.Context, c *ImageConfig) {
 			IsPresent: func() bool { return len(c.Registry.Auths) > 0 },
 			Reason:    "not supported with transfer service (also deprecated)",
 		},
-		{
-			Name:      "MaxConcurrentDownloads",
-			IsPresent: func() bool { return c.MaxConcurrentDownloads != 3 },
-			Reason:    "must be configured in transfer service plugin: plugins.\"io.containerd.transfer.v1.local\"",
-		},
```

Comment:
Not sure I fully understand why this was here, but it feels like because I'm adding a limiter to cri, this should be okay. ?

Reply:
This is here so that when a user has explicitly configured this, it will get used. If this is not here, then CRI will use the transfer service which may not have this configured. A user should move this config to the transfer service or use the defaults. You should just leave this here though.

Reply:
woops then, reverted.